cmd/prometheus/main.go

```diff
@@ -1188,6 +1192,7 @@ func (rm *readyScrapeManager) Get() (*scrape.Manager, error) {
 // This is required as tsdb.Option fields are unit agnostic (time).
 type tsdbOptions struct {
 	WALSegmentSize         units.Base2Bytes
+	MaxChunkSize           units.Base2Bytes
```

Comment:
ChunkSize is confusing, you actually mean the block chunks file size.
```suggestion
	MaxBlockChunkSegmentSize           units.Base2Bytes
```